typings/windows-process-tree.d.ts

```diff
@@ -6,14 +6,16 @@
 declare module 'windows-process-tree' {
   export enum ProcessDataFlag {
     None = 0,
-    Memory = 1
+    Memory = 1,
+    CommandLine = 2
   }
 
   export interface IProcessInfo {
     pid: number;
     ppid: number;
     name: string;
     memory?: number;
+    commandLine?: string;
```

Comment:
Let's add a note here about the 512 maximum, you don't need to explain it, just state that it's cut off.